package.json

```diff
@@ -7,13 +7,13 @@
     "build:analyze": "ANALYZE_BUNDLE=true yarn build:single",
     "build:demo": "lerna run build:demo --stream",
     "build:single": "utils/scripts/scoped-npm-command.js --script build",
-    "format": "yarn format:package_json && yarn format:js && yarn format:md",
-    "format:js": "prettier --loglevel warn --write 'packages/**/!(dist|umd)/*.{js,ts,tsx}' && prettier --loglevel warn --write utils/**/*.{js,ts,tsx}",
-    "format:md": "prettier --loglevel warn --write 'packages/**/!(dist)/!(CHANGELOG)*.md'",
-    "format:package_json": "prettier-package-json --write && lerna exec -- prettier-package-json --write",
+    "format": "prettier-package-json --write && yarn format:package_json --write && yarn format:js --write && yarn format:md --write",
+    "format:js": "prettier --loglevel warn '{packages,utils}/**/*.{js,ts,tsx}' '!packages/**/dist/**'",
```

Comment:
Yikes, [node globbing](https://github.com/isaacs/node-glob/blob/master/README.md#glob-primer) works different on CI linux than it does on my local machine. Thankfully, I was able to use CircleCI's SSH to make sure everything is behaving as expected.